addons/stock/models/stock_move_line.py

```diff
@@ -235,6 +235,10 @@ def write(self, vals):
             if key in vals:
                 updates[key] = self.env[model].browse(vals[key])
 
+        if updates.get('result_package_id'):
```

Comment:
you should probably use 
```
if 'result_package_id' in updates:
```
to handle False value in result_package_id

Reply:
Done

Reply:
Ughh, had to make a few extra changes to support this change due to package level shenanigans.